pipeline/transform/user_interests_test.py

```diff
@@ -4,7 +4,7 @@
 from pipeline.types import Interest, RawUserInterests, User
 
 
-def test_example():
+def test_example_1():
```

Comment:
Can you change this name to describe that the test is checking for? Maybe `test_add_interests`.